src/ng/services/table.spec.js

```diff
@@ -0,0 +1,92 @@
+import Table from './table';
+import angular from 'angular'; //necessary to test CellDom model property
+
+const myWindow = window.open("", "MsgWindow", "width=100,height=50");
+function htmlToElement(html) {
+	const template = myWindow.document.createElement('template');
+	template.innerHTML = html;
+	return template.content.firstChild;
+}
+
+describe('Table', () => {
+	const table = '<table tabindex=\"0\"><thead><tr><th><label aria-label=\"Last Name\">' +
```

Comment:
I'm not big fun of testing with dom elements... At least why we need all this attributes? tabindex=0 arai-label? Please remove it